atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -79,7 +82,8 @@ private AtlasDbConstants() {
             SCRUB_TABLE,
             NAMESPACE_TABLE,
             PARTITION_MAP_TABLE,
-            PERSISTED_LOCKS_TABLE);
+            PERSISTED_LOCKS_TABLE,
+            SWEEP_PROGRESS_TABLE);
```

Comment:
why not also hide `sweep.priority`?

Reply:
I think we do want to sweep `sweep.priority` as it can grow unbounded (sweep doesn't sweep hidden tables). Though quite weirdly this does mean that people can write bogus values into the priority table.

Reply:
for SA: I did a code dive to see what exactly happens if a table is hidden:
- you can't put or get from it via a normal TM (without hiddenTableAccess)
- it is not swept by the background sweeper
- the cassandra verifier ignores it when performing table name sanity checks